tools/ui-components/package.json

```diff
@@ -22,7 +22,7 @@
     "@babel/preset-typescript": "7.16.0",
     "@types/jest": "27.0.2",
     "@types/node": "16.11.6",
-    "@types/react": "17.0.33",
+    "@types/react": "17.0.34",
```

Comment:
I did not think this would be possible until we upgraded `react` to `>=17`

Reply:
Unless it causes a type error, there's nothing in the CI to stop it.  It's just a bad idea.